config/README.md

```diff
@@ -1,179 +1,305 @@
-# Config
+# Configure a package
 
-## Purpose
+The `config-package.py` is the script that configures a given repository
+to follow the standards agreed upon by the Plone community.
 
-Bring the configuration of the Plone packages into a common state and keep it there.
+## Quick start
 
-## Contents
+To configure a repository run the following commands:
 
-The `default` directory contains the following files:
+```shell
+python3.11 -m venv venv
```

Comment:
It just occurred to me that we should perhaps not refer to an explicit version of Python 3, but instead do the following, which is more common and would require less maintenance to document. Thoughts?

```suggestion
python3 -m venv venv
```